dbt/models/marts/events/github_metrics.sql

```diff
@@ -0,0 +1,88 @@
+{# 
+  Summary GitHub metrics for a project:
+    - First commit date
+    - Total repositories
+    - Total stars
+    - Total forks
+    - Total contributors
+    - Total contributors in the last 6 months
+    - Average active developers over the last 6 months
+    - Total commits over the last 6 months
+    - Total issues opened over the last 6 months
+    - Total issues closed over the last 6 months
+    - Total pull requests opened over the last 6 months
+    - Total pull requests merged over the last 6 months
+#}
+
+-- CTE for aggregating stars, forks, and repository counts by project 
+WITH StarsForksRepos AS (
```

Comment:
```suggestion
WITH stars_forks_repos AS (
```

For CTE's we're using snake case. So I would do that here